src/DynamoCore/Library/TypedParameter.cs

```diff
@@ -39,7 +39,7 @@ public string Summary
                 // once.
                 // 
                 return summary ?? (summary = ((pathManager != null)
-                    ? this.GetDescription(pathManager) : string.Empty));
+                    ? this.GetDescription() : string.Empty));
```

Comment:
It seems like the whole `IPathManager pathManager` can be removed from `class TypedParameter` :)